engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -254,6 +254,16 @@ FLUTTER_DARWIN_EXPORT
  */
 @property(nonatomic, readonly) BOOL engineAllowHeadlessExecution;
 
+/**
+ * Controls whether the created view can be sized based on its content.
+ * When set to `YES`, the FlutterView will be the same size as the outermost widget.
+ * Cannot be used with unbounded height widgets, such as Scaffold.
+ * This property is intended to be used with Add-to-App scenarios.
+ *
+ * Default is `NO`.
+ */
+@property(nonatomic, getter=isAutoResizable) BOOL autoResizable;
```

Comment:
no action required: This will be a nice place to link to the docs you are going to write as well, friendly reminder to circle back.